source/math/FloatComplexVectorList.ooc

```diff
@@ -30,21 +30,26 @@ FloatComplexVectorList: class extends VectorList<FloatComplex> {
 		super(other _vector)
 		this _count = other count
 	}
+	init: func ~withValue (capacity: Int, value: FloatComplex) {
+		super(capacity)
+		for (i in 0 .. capacity)
+			this add(value)
+	}
 	toVectorList: func -> VectorList<FloatComplex> {
 		result := VectorList<FloatComplex> new()
 		result _vector = this _vector
 		result _count = this _count
 		result
 	}
-	sum: func -> FloatComplex {
-		result := FloatComplex new()
-		for (i in 0 .. this _count)
-			result = result + this[i]
-		result
-	}
-	mean: func -> FloatComplex {
-		sum() / this _count
+	sum: FloatComplex {
+		get {
+			result := FloatComplex new()
+			for (i in 0 .. this _count)
+				result = result + this[i]
+			result
+		}
```

Comment:
My plan is to go through and clean up all the math classes & covers soon enough.